samples/helloworld/helloworld.edl

```diff
@@ -2,7 +2,10 @@
 // Licensed under the MIT License.
 
 enclave {
-    from "openenclave/edl/syscall.edl" import *;
+    // Import necessary ocalls to support fprintf/printf.
```

Comment:
How does the developer know which syscalls to import?   Is this in the docs somewhere?